.env.rinkeby

```diff
@@ -0,0 +1,10 @@
+SERVER_DB_NAME=server_wallet_test
+SERVER_DB_USER=postgres
+
+CHAIN_NETWORK_ID=4
+RPC_ENDPOINT=https://rinkeby.infura.io/v3/${INFURA_API_KEY}
+
+NITRO_ADJUDICATOR_ADDRESS=0x1d25Bbf952781e0c75d248904B5422FeE9FcA558
+ETH_ASSET_HOLDER_ADDRESS=0x0d9F0670170Fab72F0f879D020f9E77590aEE642
+ERC20_ASSET_HOLDER_ADDRESS=0x59be89a0DE5cA11c74463A696128CcD80ab0182A
+ERC20_ADDRESS=0x78573871B098da14Dc33f8c155b2ddf382eABfe3
```

Comment:
Do we have to remember to keep these addresses in sync going forward? Is there a way where we can export them from `nitro-protocol/...../deployments` ?

Reply:
Yes, we would have to remember to update these. Perhaps it is not worth investing in exporting these from `nitro-protocol` given that:
1. At the moment, this env file is used for just unit tests.
1. We probably won't run unit tests against Rinkeby that often.

However, if either of the above does not hold, we may want to reconsider.